plugins/frontend/backstage-plugin-argo-cd/src/components/useAppDetails.ts

```diff
@@ -159,7 +159,7 @@ export const useAppDetails = ({
           );
           return await Promise.all(getRevisionHistroyPromises);
```

Comment:
`return await` is a redundant. The caller already has to await this.

```suggestion
          return Promise.all(getRevisionHistroyPromises)
            .then(result => result.filter(n => n))
          ;
```